src/components/Forgot/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { Form, Input, Button, Alert } from "antd";
+import firebase from "../../firebaseConfig";
+import { MailOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+
+function Forgot({ isTesting }) {
+  const { t } = useTranslation();
+  const [email, setEmail] = useState("");
+  const [successMessage, setSuccessMessage] = useState("");
+
+  const forgotPassword = (e) => {
+    firebase
+      .auth()
+      .sendPasswordResetEmail(email)
+      .then(function (user) {
+        setSuccessMessage("Please check your email...");
+      })
+      .catch((err) => {});
+  };
+
+  const layout = {
+    labelCol: { span: 4 },
+    wrapperCol: { span: 8 },
+  };
+
+  const tailLayout = {
+    wrapperCol: { offset: 8, span: 16 },
+  };
+
+  return (
+    <Form {...layout} {...tailLayout}>
+      {successMessage ? (
```

Comment:
I think this one is enough for showing sucess message, could u please delete line 41